crates/cranelift/src/compiler.rs

```diff
@@ -261,6 +261,7 @@ impl wasmtime_environ::Compiler for Compiler {
             trampolines.push(builder.trampoline(*i, &func));
         }
         builder.align_text_to(CODE_SECTION_ALIGNMENT);
+        builder.unwind_info(&*self.isa);
```

Comment:
I think this branch needs a rebase on `main`, but as a heads up when you do that `self.isa` is now also stored within the `ObjectBuilder` itself so it'll no longer need to be an argument here.